lib/components/Textarea.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+
+import Input from './Input';
+
+interface Props {
+  name: string;
+  label?: string;
+  multiline?: true;
+}
+
+type TextAreaProps = JSX.IntrinsicElements['textarea'] & Props;
+
+export default function Textarea(props: TextAreaProps) {
+  useEffect(() => {
+    console.warn(
+      'Textarea will be deprecated in the next unform release. Please use <Input multiline /> instead',
```

Comment:
Wouldn't be better to say `in unform future releases` or something like that? The way its worded, for me at least, makes it seems that at the absolute next release it will be removed, but that's not the case, right?

Or, specify better: `the next major unform release`.

Reply:
For sure! I'll be updating this.

Thanks